Framework/Shimmer.h

```diff
@@ -0,0 +1,19 @@
+/**
+ Copyright (c) 2014-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+#import <UIKit/UIKit.h>
+
+//! Project version number for Shimmer.
+FOUNDATION_EXPORT double ShimmerVersionNumber;
+
+//! Project version string for Shimmer.
+FOUNDATION_EXPORT const unsigned char ShimmerVersionString[];
+
```

Comment:
It doesn't look like you're defining these anywhere; remove them if they aren't set?